frontend/src/Components/Loader/LoaderAnimation.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./Loader.css";
```

Comment:
No css file so this can be removed.

Reply:
I need Loader.css there, i'm using loader-circle-size & loader-text-size class from there to modify the loader size.